src/modules/getData.js

```diff
@@ -14,23 +14,31 @@ const getData = async () => {
         const myMeal = data.meals[i];
         // console.log(myMeal.idMeal);
 
-        main.innerHTML += `
-              <div class="cards-container">
-                <img
-                  class="img"
-                  src="${myMeal.strMealThumb}"
-                  alt="${myMeal.strMeal}"
-                />
-                <h2 class="title">${myMeal.strMeal}</h2>
-                <span class="thumbs-up ${myMeal.idMeal}">👍</span>
-                <p class="likes">${likes} likes</p>
-                <div class="buttons">
-                  <button class="buttons__comments">comments</button>
-                  <button class="buttons__reservation">reservation</button>
-                </div>
-              </div>`;
```

Comment:
Would you mind setting this back to your code 😀, It's being used in the new feature branch created for the dynamic likes functionality. I think as a general rule, It would be better not to delete code that was already created and working ✅, this is just to prevent future bugs in the app.